Editorial updates (#5)
MicrosoftDocs/terminal: TerminalDocs/dynamic-profiles.md

![Windows Terminal dynamic profiles](./images/dynamic-profiles.png)
_Configuration: [Light Theme](./custom-terminal-gallery/frosted-glass-theme.md)_

## Installing a new shell after installing Windows Terminal

Regardless of whether a new shell is installed before or after your terminal installation, the terminal will create a new profile for the newly installed shell.

## Hide a profile

> To hide a profile from your terminal dropdown menu, add the `hidden` property to the `profile` section of your settings.json file and set it to `true`.

Reviewer: ```suggestion
To hide a profile from your terminal dropdown menu, add the `hidden` property to the profile object in your settings.json file and set it to `true`.
```
